plugin/web/extensions/location/modals/commonModalWrapper/index.tsx

```diff
@@ -15,17 +15,18 @@ const CommonModalWrapper: React.FC<Props> = ({ title, children, onModalChange })
         <>
           <HeaderWrapper>
             <Title>{title}</Title>
-            <CloseButton>
-              <Icon icon="close" onClick={onModalChange} size={16} />
+            <CloseButton type="default">
+              <Icon size={32} icon="close" color="#00000073" onClick={onModalChange} />
```

Comment:
Please rename the prop `onModalChange` to `onModalClose`.

Reply:
done